### PR TITLE
zh-CN: glossary - code spliting

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/glossary/code_splitting/index.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+  sourceCommit: 1dc2c2e53d2f0ba0678c9967e88abeb82b2dbc84
 ---
 
 {{GlossarySidebar}}
 
-**代码分割（Code splitting）**是将代码划分为可以按需/同时加载的多个 bundles 或组件。
+**代码分割**是将 Web 应用程序依赖的代码（包括其自身代码和任何第三方依赖项）拆分为可以独立加载的单独捆绑包的做法。
 
-随着应用程序日趋复杂，或仅是对其进行简单的维护，CSS 和 JavaScripts 文件以及 bundles 的大小都会随之增加，尤其是所包含的第三方库的数量和大小的增长。为了避免下载巨大的文件，可以将脚本拆分为多个较小的文件。当前页面所需的代码能够立即加载，而另外的脚本可以在与页面或应用交互后懒加载（[lazy loaded](/zh-CN/docs/Glossary/Lazy_load)），页面性能因此提升。虽然代码的总量仍然相同（甚至可能大了几个字节），但是初次加载所需的代码数量减少了。
+这使得应用程序可以在任何给定时间点仅加载实际需要的代码，并根据需要加载其他捆绑包。这种方法用于改善应用程序的性能，特别是在初始加载时。
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/code_splitting/index.md:12]:
```suggestion
这样做，让应用程序可以在任何给定时间点仅加载实际需要的代码，并根据需要加载其他捆绑包，可以用于改善应用程序的性能，特别是可以提升初始加载时的性能。
```

File: files/zh-cn/glossary/code_splitting/index.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
-代码分割是由 [Webpack](https://webpack.js.org/) 和 [Browserify](https://browserify.org/) 等打包工具所支持的一项功能，这些打包工具能够创建在运行时动态加载的多个 bundles。
+代码分割是像 [Webpack](https://webpack.js.org/) 和 [Browserify](https://browserify.org/) 这样的打包工具的功能。这些打包工具可以创建多个可以在运行时动态加载捆绑包。
 
-## 另请参见
+## 参见
 
-- Bundling
 - [Lazy loading](/zh-CN/docs/Web/Performance/Lazy_loading)
```
Comment (PassionPenguin) [inline files/zh-cn/glossary/code_splitting/index.md:18]:
```suggestion
- [延迟加载](/zh-CN/docs/Web/Performance/Lazy_loading)
```

File: files/zh-cn/glossary/code_splitting/index.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 {{GlossarySidebar}}
 
-**代码分割（Code splitting）**是将代码划分为可以按需/同时加载的多个 bundles 或组件。
+**代码分割**是将 Web 应用程序依赖的代码（包括其自身代码和任何第三方依赖项）拆分为可以独立加载的单独捆绑包的做法。
 
-随着应用程序日趋复杂，或仅是对其进行简单的维护，CSS 和 JavaScripts 文件以及 bundles 的大小都会随之增加，尤其是所包含的第三方库的数量和大小的增长。为了避免下载巨大的文件，可以将脚本拆分为多个较小的文件。当前页面所需的代码能够立即加载，而另外的脚本可以在与页面或应用交互后懒加载（[lazy loaded](/zh-CN/docs/Glossary/Lazy_load)），页面性能因此提升。虽然代码的总量仍然相同（甚至可能大了几个字节），但是初次加载所需的代码数量减少了。
+这样做，让应用程序可以在任何给定时间点仅加载实际需要的代码，并根据需要加载其他捆绑包，可以用于改善应用程序的性能，特别是可以提升初始加载时的性能。
 
-代码分割是由 [Webpack](https://webpack.js.org/) 和 [Browserify](https://browserify.org/) 等打包工具所支持的一项功能，这些打包工具能够创建在运行时动态加载的多个 bundles。
+代码分割是像 [Webpack](https://webpack.js.org/) 和 [Browserify](https://browserify.org/) 这样的打包工具的功能。这些打包工具可以创建多个可以在运行时动态加载捆绑包。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/code_splitting/index.md:14]:
```suggestion
代码分割是像 [Webpack](https://webpack.js.org/) 和 [Browserify](https://browserify.org/) 这样的打包工具的特性。这些打包工具可以创建多个可以在运行时动态加载的捆绑包。
```

File: files/zh-cn/glossary/code_splitting/index.md [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
 
-## 另请参见
+## 参见
 
-- Bundling
-- [Lazy loading](/zh-CN/docs/Web/Performance/Lazy_loading)
+- [延迟加载](/zh-CN/docs/Web/Performance/Lazy_loading)
 - [HTTP/2](/zh-CN/docs/Glossary/HTTP_2)
 - [Tree shaking](/zh-CN/docs/Glossary/Tree_shaking)
```
Comment (jasonren0403) [inline files/zh-cn/glossary/code_splitting/index.md:20]: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#javascript

```suggestion
- [摇树优化](/zh-CN/docs/Glossary/Tree_shaking)
```

File: files/zh-cn/glossary/code_splitting/index.md [hunk 5]
```diff
@@ -1,19 +1,20 @@
 ---
 title: 代码分割
 slug: Glossary/Code_splitting
+l10n:
+  sourceCommit: 1dc2c2e53d2f0ba0678c9967e88abeb82b2dbc84
 ---
 
 {{GlossarySidebar}}
```
Comment (jasonren0403) [inline files/zh-cn/glossary/code_splitting/index.md:9]: L10 和 L12 是同一段内容，可以放在同一行

File: files/zh-cn/glossary/code_splitting/index.md [hunk 6]
```diff
@@ -1,19 +1,20 @@
 ---
 title: 代码分割
 slug: Glossary/Code_splitting
+l10n:
+  sourceCommit: 1dc2c2e53d2f0ba0678c9967e88abeb82b2dbc84
```
Comment (jasonren0403) [inline files/zh-cn/glossary/code_splitting/index.md:5]: 该更新了

```suggestion
  sourceCommit: d821201936ea3f67634381b2086998ebfc29f39c
```

**Review by jasonren0403 — COMMENTED** [review]
一些修改建议

File: files/zh-cn/glossary/code_splitting/index.md (hunk 6 above)
Comment (PassionPenguin) [reply files/zh-cn/glossary/code_splitting/index.md:5]: oops, thx!

the pr was just merged yesterday, and did forget to update this…

sorry!

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**